packages/lib-webpack/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@openshift/dynamic-plugin-sdk-webpack",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
We bump the bugfix semver number since there were no changes in package dependencies and the changelog doesn't warrant a minor version bump either.